packages/generator/src/react/context.ts

```diff
@@ -17,7 +17,34 @@ export class DefaultImportBuilder {
     }
 
     unique() {
-        return this.imports // TODO unique
+
+//If we have import {A} from 'pkg1';
+        //   import {A,B} from 'pkg2'  -- it will be deleted  -- For having all symbols we have to take symbols and create new imports.
+        const uniqueNamedImports:ts.ImportSpecifier[] = []
```

Comment:
Just use `Map<>` rather then using too many nested loops.

Reply:
Ok. I will try to do second subtask also like that and will send you in next days